api/cluster/templater_test.go

```diff
@@ -92,6 +93,20 @@ var (
 		},
 	}
 
+	tfLivenessProbe = &corev1.Probe{
```

Comment:
Why is this variable name prefixed with `tf`?

Reply:
This one only used for tensorflow